internal/private/private.go

```diff
@@ -154,6 +154,8 @@ type BlobChunkAccessor interface {
 	// The specified chunks must be not overlapping and sorted by their offset.
 	// The readers must be fully consumed, in the order they are returned, before blocking
 	// to read the next chunk.
+	// If the Length for the last chunk is set to math.MaxUint64, then it
+	// fully fetches the remaining data from the offset to the end of the blob.
```

Comment:
(Documenting it in the `ImageSourceChunk.Length` field might be helpful, and perhaps sufficient.)